internal/fnruntime/container.go

```diff
@@ -230,6 +230,14 @@ func (f *ContainerFn) checkImageExistence() bool {
 	return false
 }
 
+// AddDefaultImagePathPrefix adds default gcr.io/kpt-fn/ path prefix to image if only image name is specified
+func AddDefaultImagePathPrefix(image string) string {
+	if !strings.Contains(image, "/") {
```

Comment:
how do you handle docker hub images e.g. https://hub.docker.com/_/busybox

Reply:
Users should specify full path e.g. `docker.io/library/busybox:latest`.

docker cli prepends `docker.io/library/` by default
kpt cli prepends `gcr.io/kpt-fn/` by default